src/Security/SslContext.cs

```diff
@@ -641,22 +641,22 @@ public int SetOcspResponse (NSData response)
 			return SSLSetOCSPResponse (Handle, response.Handle);
 		}
 
-		[iOS (11,0)][TV (11,0)][Watch (4,0)][Mac (10,13)]
+		[iOS (11,0)][TV (11,0)][Watch (4,0)][NoMac] //[Mac (10,13)] Apple forgot to export SSLSetALPNProtocols. https://bugs.swift.org/browse/SR-6131
```

Comment:
it's not a generated file so you need to
`#if !MONOMAC`
over the code (so the API is not included in XM)

Reply:
Whoops you are absolutely right